module/Application/view/partial/main-nav.phtml

```diff
@@ -35,48 +35,13 @@ $lang = $this->plugin('translate')->getTranslator()->getLocale();
                     </ul>
                 </li>
                 <?php if (null === $this->identity()): ?>
-                    <li class="visible-xs pull-left">
+                    <li class="pull-right">
                         <a href="<?= $this->url('user') ?>">
                             <span class="fas fa-user"></span>
                             <span class="sr-only"><?= $this->translate('Login') ?></span>
```

Comment:
This is for screen readers, however, as the dropdown has been removed it is no longer needed, as it will read `Login` twice. You might want to consider removing it.

Reply:
I was referring to line 41, but apparently the comment changed itself.

Reply:
yess, true!